node-swc/__tests__/minify/issue_6996_test.mjs

```diff
@@ -0,0 +1,15 @@
+import swc from "../../..";
+
+it("should not throw when keep_fnames is on", async () => {
+    async function minify() {
+        const { code } = await swc.transform('function Foo() {}', {
+            jsc: {
+                minify: {
+                    keep_fnames: true,
+                },
+            },
+        });
+        return code;
+    }
+    await expect(minify()).resolves.toEqual('function Foo() {}\n');
+});
```

Comment:
it seems binding tests is done with published swc_core. I've changed local bindings/binding_core_node/Cargo.toml to have this test case covered, but the CI might fail because of this. Should the Cargo.toml be updated as well?